### PR TITLE
enhancement: home responsiveness

### PR DESCRIPTION
**Pull request by uzuriel**
- Hides history section when no data; appears when there's data
- Hides PowerBytes in navbar when below 480px in dev width
- Resizes all text varying on screen sizes
- Footer's always on the end of the screen
- Resized minimum width of dynamictable for better visualization
- History table shows recent 5 data

### CODE REVIEW

File: src/components/NavBar.jsx
Comment (chaaals) [inline src/components/NavBar.jsx:1]: Pa-remove ng `Upload CSV` button, thanks!

**Review by chaaals — CHANGES_REQUESTED** [review]
lgtm, pa-include nalang nitong isang change

File: src/components/NavBar.jsx
Comment (uzuriel) [reply src/components/NavBar.jsx:1]: > Pa-remove ng `Upload CSV` button, thanks!

as a whole or for phone/tablets?

Comment (chaaals) [reply src/components/NavBar.jsx:1]: @uzuriel as a whole, no need na siya sa navbar

**Review by chaaals — APPROVED** [review]

**Pull request merged**